buffalo/cmd/filetests/generate_resource_test.json

```diff
@@ -0,0 +1,15 @@
+[{
+    "path": "actions/college_books_test.go",
+    "contains": [
+      "func (as *ActionSuite) TableChange(table string, count int, fn func())",
```

Comment:
@alphonse927 in here we should check for the struct we're using when generating the resource tests.